### DISCUSSION TITLE
Can I specify my custom option in pytest.ini like this?

### DISCUSSION THREAD

**Discussion by ws1088** [pytest-dev/pytest#8604] [Q&A] [answered]
Hi all,
If I add an new option (e.g. `--my-opt`), can I put it in `pytest.ini` like:
```
[pytest]
; case 1: this will work
addopts = --my-opt abc

; case 2: this will not work
my_opt = abc

; case3: this will not work
my-opt = abc
```
Neither `case2` nor `case3` would work... Or is there a correct way (`case4`)?
Thanks

**Comment by The-Compiler** [comment, marked as answer]
Commandline arguments and ini options are separate - if you want an `.ini` option, you will need to use [`addini`](https://docs.pytest.org/en/6.2.x/reference.html#pytest.config.argparsing.Parser.addini) and `getini` instead.

**Reply by ws1088** [reply]
thanks for the help.
